src/components/scopedVariables/SavedVariables.tsx

```diff
@@ -30,51 +30,64 @@ export default function SavedVariablesView({
     const [variablesList, setVariablesList] = useState<VariableType[]>([])
     const [showEditView, setShowEditView] = useState<boolean>(false)
     // No need to make it a state since editor here is read only and we don't need to update it
-    let scopedVariablesYAML = parseIntoYAMLString(scopedVariablesData)
+    const scopedVariablesYAML = parseIntoYAMLString(scopedVariablesData)
 
     const { status, progress, fileData, abortRead, readFile } = useFileReader()
 
     useEffect(() => {
         if (status?.status == null) {
-            const variables = scopedVariablesData?.spec?.map((variable) => {
-                return {
-                    name: variable.name,
-                    description: variable.description,
-                }
-            })
+            const variables = scopedVariablesData?.spec?.map((variable) => ({
```

Comment:
We should move this check in the if statement and we can omit the next if(variable) check

Reply:
Done